app/services/complex_cases/importer.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module ComplexCases
+  class Importer
+    COMPLEX_CASES = [
+      { key: 'segregated', title: 'Segregated prisoners' },
```

Comment:
I wonder if we could use the same keys/terms we have used within the assessment sections to identify these scenarios. We worked a lot on those terms during early prototyping with our content designer and and to keep consistency in how we referred to scenarios across different organisations (Police/Prison/SCH/etc). This [spreadsheet](https://docs.google.com/spreadsheets/d/1f0JlJ6T1cL7fHxVQ2LOAokPacNPoFk_f35Pzucyc3UI/edit?ts=5e21635f) is a useful reference of all the mappings and terms we created across all the different systems.

For example I think we could swap these for some existing terms:
`segregated` -> `hold_separately`
`acct` -> `self_harm`
`mental` -> `mental_health_issue`
`drugs` -> `under_drug_treatment`

cc/ @Andy-Millar

Reply:
Worth noting, the entire page is criteria for the allocation. Just to stress, we are not doing anything clever at the moment with filtering prisoners - just communicating to the OCAs what is the criteria for people to add to allocation. 

May be worth segmenting the criteria as follows:
- criteria-category
- criteria-sentence-length
- criteria-segregated
- criteria-self-harm AND criteria-acct [note: checkbox ticks both, but maybe later we want to seperate]
- criteria-mental-health
- criteria-integrated-drug-treatment-system
- fulfillment-status
- criteria-other

Hope this is helpful!

Reply:
Thanks both, I'll amend the keys to match the suggestions used elsewhere for now.

Reply:
@smoothcontract after chatting with Andy again it does feel like there is a similarity between `sentence_length`, `prison_category` and `complex_cases`. And with the knowledge that these may change a lot in the future I wonder if it is worth thinking about mapping that relationship to this model? For example, having a `criteria` attribute or relationship that contains all the possible filtering criteria.

Reply:
I really want to avoid doing that now - it changes a fair bit of existing work and makes it challenging to deliver the stories for this sprint. I'd prefer to tackle that change at some point in the future when we revisit filtering, after getting a better idea of how the data is used.

Reply:
Happy to leave if for this sprint. I think we have a good understanding now that there is a concept of "criteria" on an allocation so when we look at filtering we can have a think about how we model these on the data layer.